### PR TITLE
kibana_plugin: fixed remove call + run_command with list instead of str

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Added missing arguments to `remove_plugin()` when `state=present force=true`.

While at it, when calling `run_command()` it was converting a list to str, so changed it to keep the list.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
kibana_plugin

### CODE REVIEW

**Comment by felixfontein** [comment]
The module does things like `cmd_args.append("--url %s" % url)`, which make this a non-working change. These need to be modified to `cmd_args.extend(["--ur", url])`.

**Comment by russoz** [comment]
Oopsie. Fixed, sent.

File: changelogs/fragments/2143-kibana_plugin-fixed-function-calls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - kibana_plugin - added missing parameters to ``remove_plugin`` when using ``state=present force=true`` (https://github.com/ansible-collections/community.general/pull/2143).
```
Comment (felixfontein) [inline changelogs/fragments/2143-kibana_plugin-fixed-function-calls.yml:2]:
```suggestion
  - kibana_plugin - added missing parameters to ``remove_plugin`` when using ``state=present force=true``, and fix potential quoting errors when invoking ``kibana`` (https://github.com/ansible-collections/community.general/pull/2143).
```

**Review by felixfontein — COMMENTED** [review]
Besides that, looks good.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for the improvements!

**Comment by felixfontein** [comment]
Manual backport to stable-1 in #2154.

**Pull request merged**